backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/AccountRequestController.java

```diff
@@ -98,7 +98,7 @@ public AccountResponse submitOrganizationAccountRequest(
             "This email address is already associated with a SimpleReport user.");
       } else {
         throw new BadRequestException(
-            "An unknown error occured when creating this organization in Okta.");
+            "An unknown error occurred when creating this organization in Okta.");
```

Comment:
I think you'll also need to update [utils.tsx](https://github.com/CDCgov/prime-simplereport/blob/5c7eddb272de31c3cbbf4c839b4ec09cfce1705e/frontend/src/app/signUp/Organization/utils.tsx) on the frontend to make sure this message is being parsed correctly.

Reply:
Oh, good catch!  I didn't realize these messages were coupled to the frontend.